Changelog.md

```diff
@@ -75,49 +80,19 @@
 
 ## Ignition Gazebo 2.x
 
-
-### Ignition Gazebo 2.20.0 (2020-06-09)
```

Comment:
Oops, maybe some bad conflict resolution?

Reply:
should be fixed now. I think my ign-gazebo3 branch was not up-to-date